app/src/main/java/com/infomaniak/mail/ui/sync/SyncAutoConfigViewModel.kt

```diff
@@ -41,24 +41,22 @@ class SyncAutoConfigViewModel @Inject constructor(
 ) : AndroidViewModel(application) {
 
     private val ioCoroutineContext = viewModelScope.coroutineContext(ioDispatcher)
-
+    private var credentialsJob: Job? = null
     val snackBarManager by lazy { SnackBarManager() }
 
-    private var credentialsJob: Job? = null
+    fun isSyncAppInstalled(): Boolean = runCatching {
+        context.packageManager.getPackageInfo(SYNC_PACKAGE, PackageManager.GET_ACTIVITIES)
+        true
+    }.getOrElse {
+        false
+    }
 
+    // TODO: Add all Matomo in SyncAutoConfig feature
```

Comment:
Maybe you could direclty put the navigation tracking in this PR as it's the one that fragments and navigation